_std/macros/factions.dm

```diff
@@ -1,22 +1,34 @@
 //Faction bitmasks, for setting what teams mobs/critters are on. Barebones atm.
 
+/// Faction which is not targeted by default
+#define FACTION_NEUTRAL			(1<<0)
+/// Generic faction for groups you don't want killing eachother
+#define FACTION_GENERIC			(1<<1)
```

Comment:
that doesn't make sense. this would make more sense as a 'default faction' or similar.